prometheus/config/args/install.sls

```diff
@@ -64,22 +56,35 @@ prometheus-config-args-{{ name }}-collector-textfile-directory:
     - value: {{ args.pop('collector.textfile.directory') }}
     - watch_in:
       - service: prometheus-service-running-{{ name }}-service-running
-    - require:
-      - file: prometheus-config-file-args-file-directory
 
                 {%- endif %}
                 {%- if 'storage.tsdb.path' in args.keys() %}
 
-prometheus-config-args-{{ name }}-{{ key }}:
+prometheus-config-args-{{ name }}-storage-tsdb-path:
```

Comment:
Theses states are only executed for FreeBSD 
Is `storage-tsdb-path` only relevant to FreeBSD?  
I cannot remember.

Reply:
This is just the sysrc setting. The actual directory is handled in line 36: `prometheus-config-args-{{ name }}-data-dir`.